### PR TITLE
[Feature:InstructorUI] Add links Office Hour Queue docs

### PR DESCRIPTION
**Pull request by bmcutler**
Add links to the submitty.org student and grader documentation.
(Also, fix minor spelling mistake in html)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -348,7 +348,7 @@ jobs:
         - sudo bash .setup/travis/setup.sh
         - sudo bash .setup/travis/setup_test_suite.sh
       script:
-        - sudo python3 /usr/local/submitty/test_suite/integrationTests/run.py
+        - travis_wait 45 sudo python3 /usr/local/submitty/test_suite/integrationTests/run.py
```
Comment (elihschiff) [inline .travis.yml]: Did you mean to add this? It seems like this should not be in this pr

Comment (bmcutler) [reply .travis.yml]: Trying a thing to see if it fixes the 10m0s timeout bug
(Derek discovered this magic Travis hack)

Comment (elihschiff) [reply .travis.yml]: Ok, I just wanted to make sure this code was not added by mistake.

**Comment by bmcutler** [comment]
@elihschiff can you make sure I didn't break anything adding the word 'access' in front of code?

**Comment by elihschiff** [comment]
That change wont break anything other than maybe a test, which if it does will be a really easy fix.

**Pull request merged**